### ISSUE TITLE
Using Object.entries & Object.values should be flagged for old Android & iOS Safari browsers

### ISSUE THREAD

**Issue by playpauseandstop** [amilajack/eslint-plugin-compat#231]
Hi,

First of all, thanks a lot for all your work to provide this great ESLint plugin. It helps me a lot. But recently I ran into strange issue with the plugin, which would like to discuss.

I'm using,

- OS: macOS 10.14.4
- node.js 12.1.0
- eslint-plugin-compat: 3.1.1

`package.json` defines next browsers list,

```json
  "browserslist": [
    "Android >= 4",
    "iOS >= 7"
  ]
```

But the code, which contains,

```javascript
Object.values({})
Object.entries({})
```

doesn't flagged as error.

I'm temporarily fix this by adding `"Safari >= 8"` into the browsers list, but still interested in more appropriate fix.

----

caniuse website for sure shown, that `Object.entries` & `Object.values` don't supported by iOS 7 & Android 4,

- https://caniuse.com/#search=Object.entries
- https://caniuse.com/#search=Object.values

But after [adding the test](https://github.com/playpauseandstop/eslint-plugin-compat/commit/a225b36c8df74b7bfb640ac859eb2d576e2c8733), `yarn test` failed locally for me with:

<details>
<summary>Test Results</summary>
```
 FAIL  test/e2e.spec.js
  ● compat › invalid › Object.entries({})

    assert.strictEqual(received, expected)

    Expected value to strictly be equal to:
      1
    Received:
      0
    
    Message:
      Should have 1 error but had 0: []

      at testInvalidTemplate (node_modules/eslint/lib/testers/rule-tester.js:462:24)
      at Object.<anonymous> (node_modules/eslint/lib/testers/rule-tester.js:592:25)

  ● compat › invalid › Object.values({})

    assert.strictEqual(received, expected)

    Expected value to strictly be equal to:
      1
    Received:
      0
    
    Message:
      Should have 1 error but had 0: []

      at testInvalidTemplate (node_modules/eslint/lib/testers/rule-tester.js:462:24)
      at Object.<anonymous> (node_modules/eslint/lib/testers/rule-tester.js:592:25)
```
</details>

Any ideas on why this happen in first place and how to fix this behavior?

**Comment by brettz9** [comment]
I think this should be closed in favor of the more generic #206 (letting that apply to all ES features).
